### PR TITLE
Instructions for CD on render.com

### PR DESCRIPTION
**Pull request by jon-bell**
It would be best if at least two TAs can independently follow these instructions to deploy their own fork. If there are any issues, please push new commits directly to this branch to correct them (as opposed to adding comments requesting changes). This link can be used to create new repositories with the starter codebase:  https://classroom.github.com/a/7Ax0k2SR

### CODE REVIEW

**Comment by mwand** [comment]
This shows up as a Tutorial instead of an Activity.

**Comment by abhutta0** [comment]
> This shows up as a Tutorial instead of an Activity.

If we move it to activities, it wont show up as a link in drop downs. We will have to embed the link somewhere else.

File: tutorials/ci-cd.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+---
+
+# Continuous Development Pipelines for Covey.Town
+
+In this activity, you will configure a continuous development pipeline using [GitHub Actions](https://github.com/features/actions) and [Render.com](https://www.render.com). Our pipeline will use GitHub Actions to build and test your Covey.Town fork, and use Render.com to deploy the application.
+
+Only one member of each team needs to do these steps - the resulting deployment will be shared by the whole team.
+
+## Pre-requisites
+There are two pre-requisites for this activity: you must have a GitHub.com repository generated following our [project forking instructions]({{ site.baseurl }}{% link Assignments/project-overview.md %}), and you must have a Twilio account.
```
Comment (sharan-parikh) [inline tutorials/ci-cd.md:16]: URL for project forking instructions needs to be updated.

Comment (sharan-parikh) [reply tutorials/ci-cd.md:16]: The instructions are okay, no changes are needed other than the above comment.

**Comment by mwand** [comment]
On Tue, Oct 24, 2023 at 9:22 AM Adeel A. Bhutta ***@***.***>
wrote:

> This shows up as a Tutorial instead of an Activity.
>
> If we move it to activities, it wont show up as a link in drop downs. We
> will have to embed the link somewhere else.
>
>
> That's what we do with all the other graded Activities  (and this one
should definitely be graded!)

**Pull request merged**